component/admin/language/en-GB/en-GB.com_localise.ini

```diff
@@ -161,6 +161,7 @@ COM_LOCALISE_CANNOT_REMOVE_DEFAULT_LANGUAGE="Localise cannot remove default lang
 
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT="Translate"
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT_DESC="Translate description"
+COM_LOCALISE_CONFIRM_TRANSLATION_SAVE="Before saving, make sure you have set the translation as 'Complete' or not in the 'Details' tab."
```

Comment:
I think this would make more sense in english if it said
+COM_LOCALISE_CONFIRM_TRANSLATION_SAVE="Before saving, make sure you have set if the translation is 'Complete' or not in the 'Details' tab.